src/StackAdmin/Azs.Compute.Admin/Module/Azs.Compute.Admin/Generated.PowerShell.Commands/SwaggerDefinitionCommands/New-QuotaObject.ps1

```diff
@@ -28,6 +28,12 @@ Changes may cause incorrect behavior and will be lost if the code is regenerated
 .PARAMETER VmScaleSetCount
     Maximum number of scale sets allowed.
 
+.PARAMETER MaxAllocationStandardManagedDisksAndSnapshots
```

Comment:
StandardManagedDiskAndSnapshotSize

Reply:
This is the model that is passed to SDK, we need to have these names here right ?
The new names are defined in New-QuotaCustomObject.

Reply:
On the client side you will only have the QuotaCustomObject.  Only inside the cmdlets will you have the SDK Quota Object.  So clients should ONLY see properties that relate to the Custom object.

Reply:
yep.